src/components/GameBoard.js

```diff
@@ -66,16 +56,15 @@ function GameBoard({ muted }) {
 		}
 
 		// generates the walls/outline of the rooms
-		for (let i = yLoc + 1; i < yLoc + h; i++) {
-			tempBoard[i][xLoc + 1] = wall;
-			tempBoard[i][xLoc + w - 1] = wall;
-		}
-		for (let j = xLoc + 1; j < xLoc + w; j++) {
-			tempBoard[yLoc + 1][j] = wall;
-			tempBoard[yLoc + h - 1][j] = wall;
-		}
+		tempBoard = createWalls(
+			tempBoard,
+			xLoc + 1,
+			xLoc + w,
+			yLoc + 1,
+			yLoc + h,
+		);
 		if (roomSize.door) {
-			tempBoard[roomSize.door.y + yLoc][roomSize.door.x + xLoc] = null;
+			tempBoard[roomSize.door.y + yLoc][roomSize.door.x + xLoc] = building;
```

Comment:
I assume we will eventually have a door tile, but I changed this to a building tile for right now. The smooth walls looked strange as grass.